solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -852,6 +815,68 @@ contract LiquidityProtection is TokenHandler, ContractRegistryClient, Reentrancy
         return outputAmount.add(_reserveAmount.mul(compN).div(compD));
     }
 
+    /**
+      * @dev returns the amount of tokens required for liquidation
+      *
+      * @param _poolToken       pool token
+      * @param _reserveToken    reserve token
+      * @param _poolAmount      pool token amount when the liquidity was added
+      * @param _targetAmount    target token amount received in return
+      * @param _isBaseAmount    true if the required amount is of the based token
```

Comment:
the required amount is in pool tokens so this comment is confusing as there's no required amount in base tokens

Reply:
Can you please fix this comment then?